test/helper.ts

```diff
@@ -33,3 +35,17 @@ export function parseAndProcess(input: string, fn: (ast: ESTree.Program) => void
   fn(ast);
   return astring(ast).trim();
 }
+
+const defaultSandbox = {
+  console,
+  module: {
+    exports: {}
+  },
+  require
+};
+
+export function virtualModule(code: string, optionsSandbox = {}) {
```

Comment:
My comment is not about the result of the merge but about the function signature of virtualModule. The defaultSandbox is fine for me, since it is defined directly above and the types are 'visible'. But the optionsSandbox could contain literaly anything.
This should not be intended behaviour and even though its just for a test (or a test helper) developers should understand whats the function signature and contract is.